velox/connectors/hive/HiveConnector.cpp

```diff
@@ -148,6 +149,9 @@ HiveDataSource::HiveDataSource(
         it != columnHandles.end(),
         "ColumnHandle is missing for output column: {}",
         outputName);
+    if (readAsLowerCase) {
+      VELOX_CHECK(!std::any_of(outputName.begin(), outputName.end(), isupper));
```

Comment:
I'm not sure if we need to check the struct type subType. @Yuhta

Reply:
If the planner is expected to lowercase subfield names, we should check them